bokehjs/src/coffee/models/renderers/glyph_renderer.coffee

```diff
@@ -16,22 +16,22 @@ class GlyphRendererView extends Renderer.View
 
     selection_glyph = @mget("selection_glyph")
     if not selection_glyph?
-      attrs = _.extend {}, glyph_attrs, @model.selection_defaults
-      selection_glyph = new (@mget("glyph")).constructor(attrs)
+      #attrs = _.extend {}, glyph_attrs, @model.selection_defaults
```

Comment:
That's a different thing. Here the problem is that e.g. `decimated_defaults` contain both fill and line defaults, when glyph can just support non or one of them (e.g. `Line`).